pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.ullink.jira</groupId>
     <artifactId>jira-slackit</artifactId>
-    <version>6-${maven.build.timestamp}</version>
+    <version>0.0.1</version>
```

Comment:
version number is going down, on purpose ?

Reply:
Fixed it was typo. 7.0.0.  Also, I removed ${maven.build.timestamp} because it is generating everytime a new jar. 
Fortunately, Atlassian UPM (Universal plugin manger) resolving correctly build with stricted versioning